src/app/guides/story/page.mdx

```diff
@@ -0,0 +1,733 @@
+export const ant = ANT.init({
+  signer: new ArweaveSigner(jwk),
+  processId: process.env.ANT_PROCESS_ID || "",
+});
+
+// Initialize the Turbo client for uploads
+export const turbo = TurboFactory.init({
+  signer: new TurboSigner(jwk)
+});
+
+// Helper function to determine content type based on file extension
+export function getContentType(fileName) {
+  const extension = fileName.split('.').pop().toLowerCase();
+  const contentTypes = {
+    'jpg': 'image/jpeg',
+    'jpeg': 'image/jpeg',
+    'png': 'image/png',
+    'gif': 'image/gif',
+    'pdf': 'application/pdf',
+    'json': 'application/json',
+    'html': 'text/html',
+    'txt': 'text/plain',
+    // Add more types as needed
+  };
+  
+  return contentTypes[extension] || 'application/octet-stream';
+}
+```
+
+## 2. Set up metadata for your IP Asset
+
+First, let's set up the IP and NFT metadata:
+
+```typescript main.ts
+import { client } from "./utils";
+import { ant, turbo, getContentType } from "./arweave-utils";
+import fs from 'fs';
+import { createHash } from "crypto";
+
+async function main() {
+  // Set up metadata for your IP asset
+  const ipMetadata = {
+    title: "Arweave IP Asset Example",
+    description: "An IP asset stored on Arweave and registered on Story Protocol",
+    image: "", // We'll update this after uploading to Arweave
+    imageHash: "",
+    mediaUrl: "", // We'll update this after uploading to Arweave
+    mediaHash: "",
+    mediaType: "image/png",
+    creators: [
+      {
+        name: "AR.IO Developer",
+        address: "0x1234567890123456789012345678901234567890", // Replace with your ETH address
+        description: "Developer integrating Arweave with Story Protocol",
+        contributionPercent: 100,
+        socialMedia: [
+          {
+            platform: "Website",
+            url: "https://ar.io",
+          },
+        ],
+      },
+    ],
+  };
+
+  // Set up metadata for your NFT
+  const nftMetadata = {
+    name: "Arweave IP Asset",
+    description: "This NFT represents ownership of an IP Asset stored on Arweave",
+    image: "", // We'll update this after uploading to Arweave
+  };
+
+  // Continue with the rest of the process...
+}
+
+main().catch(console.error);
+```
+
+## 3. Upload assets to Arweave
+
+Now let's add the functionality to upload files to Arweave:
+
+```typescript main.ts
+// ...previous code
+
+async function uploadToArweave(filePath) {
+  // Get the file name from the path
+  const fileName = filePath.split('/').pop();
+  
+  // Read the file
+  const fileData = fs.readFileSync(filePath);
+  
+  // Determine content type based on file extension
+  const contentType = getContentType(fileName);
+  
+  console.log(`Uploading ${fileName} to Arweave...`);
+  
+  // Upload the file to Arweave
+  const uploadResult = await turbo.uploadFile({
+    fileData,
+    tags: [
+      { name: 'Content-Type', value: contentType },
+      { name: 'App-Name', value: 'Story-Arweave Integration' },
```

Comment:
Should this value for 'App-Name' just be 'Story-Protocol'

The addition of "Arweave Integration" feel redundant as they're using Arweave for storage